c7n/resources/apigw.py

```diff
@@ -309,7 +313,193 @@ def augment(self, resources):
         return results
 
 
-ANNOTATION_KEY = 'c7n-matched-resource-methods'
+@resources.register('rest-vpclink')
+class RestAPI(query.QueryResourceManager):
```

Comment:
this is redefining another class with the same name in this module, while it doesn't cause an error, its not really safe or advisable, nor i think intended.

Reply:
Good Catch.  I didn't do that intentionally as I know it's not correct to name classes the same. Probably due to me reusing existing code and not changing class names in that one rest-vpclink resource. Fixed now.